src/mdtest.c

```diff
@@ -1564,7 +1561,7 @@ void summarize_results(int iterations) {
 /* Checks to see if the test setup is valid.  If it isn't, fail. */
 void valid_tests() {
 
-    if(stone_wall_timer_seconds > 0 && branch_factor > 1 || ! barriers){
+    if((stone_wall_timer_seconds > 0 && branch_factor > 1) || ! barriers){
```

Comment:
Shouldn't we also protect `stone_wall_timer_seconds > 0` and `branch_factor > 1` individually? 
For example:
if((stone_wall_timer_seconds > 0) && (branch_factor > 1) || ! barriers){

Reply:
The compiler was not complaining about those. I'll change this as well as it improves readability.